pom.xml

```diff
@@ -137,6 +137,8 @@
                         <includes>
                             <include>**/*IT.java</include>
                             <include>**/*IT.groovy</include>
+                            <include>**/*FT.java</include>
```

Comment:
do we need the rule for .java?

Reply:
I checked and we need this rule, even if all test are in groovy